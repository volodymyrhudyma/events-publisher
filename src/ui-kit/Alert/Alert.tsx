import React from 'react';
import { PrimaryText, SecondaryText, Wrapper } from './styled/Alert.styled';

interface IAlertProps {
	primaryText: string;
	secondaryText: string;
}

const Alert = ({ primaryText, secondaryText }: IAlertProps) => (
	<Wrapper>
		<PrimaryText>{primaryText}</PrimaryText>
		<SecondaryText>{secondaryText}</SecondaryText>
	</Wrapper>
);

export default Alert;
