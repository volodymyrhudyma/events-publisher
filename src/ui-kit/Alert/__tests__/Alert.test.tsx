import React from 'react';
import { shallow } from 'enzyme';
import Alert from '../Alert';

describe('Alert component', () => {
  it('should render correctly', () => {
    const component = shallow(
      <Alert primaryText="Success" secondaryText="Success" />
    );
    expect(component).toMatchSnapshot();
  });
});
