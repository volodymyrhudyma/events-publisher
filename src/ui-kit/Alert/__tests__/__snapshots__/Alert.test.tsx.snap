// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert component should render correctly 1`] = `
<styled.div>
  <styled.div>
    Success
  </styled.div>
  <styled.div>
    Success
  </styled.div>
</styled.div>
`;
